Docs: BuildInstructionsLadybird.md doesn't mention the Qt ≥ 6.9 requirement introduced in 4aba06c3d8

### Summary

I pulled and built master on Friday without issues on my trixie WSL2 setup; after pulling again over the weekend, configure now fails since [4aba06c3d8](https://github.com/LadybirdBrowser/ladybird/commit/4aba06c3d8) ("UI/Qt: Require Qt version 6.9", 2026-07-10) landed:

```
CMake Error at Build/vcpkg/scripts/buildsystems/vcpkg.cmake:910 (_find_package):
  Could not find a configuration file for package "Qt6" that is compatible
  with requested version "6.9".

    /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake, version: 6.8.2
```

`Documentation/BuildInstructionsLadybird.md` hasn't been updated to reflect this:

- The prerequisites line says "Qt6 development packages … are required" without a minimum version.
- The **Debian/Ubuntu** section instructs installing `qt6-base-dev` from the distro, but Debian 13 (trixie) ships Qt 6.8.2 and no current Ubuntu LTS ships 6.9 either, so following the instructions verbatim produces the error above. Qt 6.9+ is currently only available in Debian testing/sid and Ubuntu 25.10+.
- The **WSL2** section recommends "Ubuntu or Fedora" with the same package lists, so WSL users hit the same wall.

**Suggestion:** state the Qt ≥ 6.9 minimum in the prerequisites, and note which distro releases currently satisfy it (or point affected users at the vcpkg-provided Qt 6.10 mentioned in the commit message).

**Environment:** Debian 13 (trixie) under WSL2, master @ current

### Operating system

Linux

### Steps to reproduce

1. On Debian 13 (trixie) under WSL2, install the packages listed in the Debian/Ubuntu section of `Documentation/BuildInstructionsLadybird.md` (this installs `qt6-base-dev` 6.8.2, the newest available in trixie)
2. Clone master (any commit including or after 4aba06c3d8)
3. Run `./Meta/ladybird.py build`

### Expected behavior

Following the documented prerequisites for a listed distro produces a working configure/build, or the documentation states the minimum Qt version (≥ 6.9) so the reader knows their distro's Qt is too old before starting.

### Actual behavior

Configure fails because system Qt 6.8.2 doesn't satisfy the new `find_package(Qt6 6.9 REQUIRED)` in UI/Qt/CMakeLists.txt:

```
CMake Error at Build/vcpkg/scripts/buildsystems/vcpkg.cmake:910 (_find_package):
  Could not find a configuration file for package "Qt6" that is compatible
  with requested version "6.9".

    /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake, version: 6.8.2
```

### URL for a reduced test case

N/A

### HTML/SVG/etc. source for a reduced test case

```html
N/A
```

### Log output and (if possible) backtrace


```shell
N/A — documentation issue, no crash involved
```

### Screenshots or screen recordings

_No response_

### Build flags or config settings

_No response_

--- a/Documentation/BuildInstructionsLadybird.md
+++ b/Documentation/BuildInstructionsLadybird.md
@@ -2,7 +2,10 @@
 
 ## Build Prerequisites
 
-Qt6 development packages, nasm, additional build tools, and a C++23 capable compiler are required.
+Qt6.9+ development packages, nasm, additional build tools, and a C++23 capable compiler are required.
+
+> [!NOTE]
+> Some distributions still package a Qt6 older than 6.9; for example, Debian 13 (trixie) ships Qt 6.8 — so configuring against it fails. If your Qt6 is older than 6.9, install a newer one from a newer distribution release, or directly from the [Qt online installer](https://www.qt.io/download-open-source), and then point the build to it using `CMAKE_PREFIX_PATH`.
 
 A Rust toolchain is also required. You can install it via [rustup](https://rustup.rs/).
 

